### PR TITLE
Adding SS-2015-006

### PR DESCRIPTION
**Pull request by spekulatius**

### CODE REVIEW

File: silverstripe/framework/SS-2015-006-1.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title:     SS-2015-006: XSS In GridField print
+link:      http://www.silverstripe.org/software/download/security-releases/ss-2015-006/
+cve:       ~
+branches:
+    3.1.x:
+        time:     2015-02-12 15:55:00
```
Comment (xabbuh) [inline silverstripe/framework/SS-2015-006-1.yaml]: The date looks wrong (the blog post was published February 19th and the tag was made at 2015-02-18 23:37 UTC).

Comment (spekulatius) [inline silverstripe/framework/SS-2015-006-1.yaml]: Hello @xabbuh ,

I relying on the release date. Which blog post are you referring to? 

Spek

Comment (xabbuh) [inline silverstripe/framework/SS-2015-006-1.yaml]: @spekulatius On http://www.silverstripe.org/software/download/security-releases/ss-2015-006/, they say that the release date is 2015-02-19.

Comment (stof) [inline silverstripe/framework/SS-2015-006-1.yaml]: @xabbuh the time here should be the time at which the fix was merged, not the release date (the goal is to be able to know whether dev branches are impacted or no)

Comment (xabbuh) [inline silverstripe/framework/SS-2015-006-1.yaml]: @stof Oh indeed, I misinterpreted the contribution docs in the past. I should re-check my PRs that have been merged in the past. :(

**Pull request merged**